packages-web/utils-react-widgets/configs/webpack.config.prod.js

```diff
@@ -79,19 +97,35 @@ const previewConfig = {
         libraryTarget: "commonjs"
     },
     resolve: {
-        extensions: [".ts", ".js"]
+        extensions: [".ts", ".js", ".tsx", ".jsx"]
     },
     module: {
         rules: [
             {
-                test: /\.ts$/,
+                test: /\.tsx?$/,
                 loader: "ts-loader",
                 options: {
```

Comment:
Discuss: for compatibility should everything though ts-loader and babel?

Reply:
both :)